bot/exts/info/doc/_cog.py

```diff
@@ -340,11 +340,13 @@ async def get_command(self, ctx: commands.Context, *, symbol_name: Optional[str]
 
             if doc_embed is None:
                 error_message = await send_denial(ctx, "No documentation found for the requested symbol.")
+
                 await wait_for_deletion(error_message, (ctx.author.id,), timeout=NOT_FOUND_DELETE_DELAY)
-                with suppress(discord.NotFound):
-                    await ctx.message.delete()
-                with suppress(discord.NotFound):
-                    await error_message.delete()
+
+                if not (ctx.message.mentions or ctx.message.role_mentions):
+                    with suppress(discord.NotFound):
+                        await ctx.message.delete()
+
```

Comment:
You still need to remove `error_message`.
```suggestion
                with suppress(discord.NotFound):
                    await error_message.delete()
```

Reply:
> You still need to remove `error_message`.

That will be deleted in [L#343?](https://github.com/python-discord/bot/blob/c5ab6af7770745d69e0d1d04fc07d7b8c601f98a/bot/exts/info/doc/_cog.py#L343)